WordPress/Jetpack/Classes/NUX/New Landing Screen/Views/JetpackLandingScreenView.swift

```diff
@@ -1,20 +1,68 @@
-import SwiftUI
+import UIKit
 
-struct JetpackLandingScreenView: View {
+final class JetpackLandingScreenView: UIView {
 
     private let prompts = JetpackPromptsConfiguration.Constants.basePrompts
     private let evenColor = JetpackPromptsConfiguration.Constants.evenColor
     private let oddColor = JetpackPromptsConfiguration.Constants.oddColor
+    private let lineSpacing: CGFloat = -6
+    private let interitemSpacing: CGFloat = -2
+    private let font: UIFont = UIFont.systemFont(ofSize: 40, weight: .bold)
+    private let insets = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
 
-    var body: some View {
-        VStack(alignment: .leading, spacing: 0) {
-            ForEach(0..<prompts.count * 2, id: \.hashValue) { index in
-                JetpackPromptView(
-                    text: prompts[index % prompts.count],
-                    color: index % 2 == 0 ? evenColor : oddColor
-                )
-            }
+    init() {
+        super.init(frame: .zero)
+        self.setup()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setup() {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .leading
+        stackView.axis = .vertical
+        stackView.spacing = self.interitemSpacing
+        for index in 0..<prompts.count * 2 {
+            let view = self.promptView(
+                text: prompts[index % prompts.count],
+                color: index % 2 == 0 ? evenColor : oddColor
+            )
+            stackView.addArrangedSubview(view)
         }
-        .frame(maxWidth: .infinity, alignment: .leading)
+        self.addSubview(stackView)
+        self.pinSubviewToAllEdges(stackView, insets: insets)
+    }
+
+    private func promptView(text: String, color: UIColor) -> UIView {
+        let textView = UITextView()
+        textView.accessibilityElementsHidden = true
+        textView.isSelectable = false
+        textView.isEditable = false
+        textView.isScrollEnabled = false
+        textView.textContainer.maximumNumberOfLines = 0
+        textView.textContainer.lineFragmentPadding = 0
+        textView.textContainerInset = .zero
+        textView.backgroundColor = .clear
+
+        let attributedString = NSMutableAttributedString(string: text)
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineSpacing = lineSpacing
+        let attributes: [NSAttributedString.Key: Any] = [
+            .paragraphStyle: paragraphStyle,
+            .foregroundColor: color,
+            .font: font
+        ]
+        attributedString.addAttributes(attributes, range: NSMakeRange(0, text.count))
```

Comment:
This works fine for the current prompts, but may break things if "wider" characters are used in the future - like emojis. We had a [similar situation](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/WordPress/Classes/ViewRelated/Site%20Creation/Site%20Name/SiteNameView.swift#L158) with the Site Name project.

[This SO answer](https://stackoverflow.com/a/57525702) has more info, with an alternative as well.

```suggestion
        attributedString.addAttributes(attributes, range: NSMakeRange(0, text.utf16.count))
```

Reply:
Good catch @twstokes! I committed your suggestion.